hw09-jdbc/src/main/java/ru/otus/hw09/jdbc/ReflectionHelper.java

```diff
@@ -0,0 +1,102 @@
+package ru.otus.hw09.jdbc;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ru.otus.hw09.orm.Id;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ReflectionHelper {
```

Comment:
```suggestion
public class SqlQueryHelper {
```

В `ReflectionHelper` вы вынесли больше, чем просто работа с рефлексией. У вас получился уже не `ReflectionHelper`, а `SqlQueryHelper`